cmd/kogito/command/install/data_index.go

```diff
@@ -165,6 +186,7 @@ func (i *installDataIndexCommand) Exec(cmd *cobra.Command, args []string) error
 		i.flags.infinispan.Credentials.UsernameKey = defaultInfinispanUsernameKey
 		i.flags.infinispan.Credentials.PasswordKey = defaultInfinispanPasswordKey
 		i.flags.infinispan.UseAuth = true
+		i.flags.infinispan.UseKogitoInfra = false
```

Comment:
What is the use case here ? Why would we change this value ?

Reply:
If you provide your own authentication options, this means that you're using another Infinispan provider/installation, like an on premise Infinispan installation. Most use cases will use our `KogitoInfra`.

Reply:
right... missed that in arguments checks ...